index.html

```diff
@@ -850,19 +851,14 @@
             </li>
             <li>Let <em>P</em> be a new <a>Promise</a>.
             </li>
-            <li>Return <em>P</em>.
+            <li>Return <em>P</em>, but continue running these steps in
+            parallel.
```

Comment:
The following steps need to be run in order they are presented ... maybe drop the "in parallel" bit?